terraform/modules/eventbridge/main.tf

```diff
@@ -0,0 +1,36 @@
+resource "aws_eventbridge_rule" "module_event" {
+  name                = var.name
```

Comment:
this will need to use the "iam_prefix" like other modules so that we can create these resources in the lower tiers